galata/test/jupyterlab/collapsible-headings.test.ts

```diff
@@ -111,3 +111,188 @@ test.describe('Collapsible Headings; no_showHCB', () => {
     ).toMatchSnapshot('no_showHCB_expand_heading_via_collapser.png');
   });
 });
+
+async function populateNotebook2(page: IJupyterLabPageFixture) {
+  await page.notebook.setCell(0, 'markdown', '# Heading 1');
+  await page.notebook.addCell('code', '1+1');
+  await page.notebook.addCell('markdown', '## Heading 1.1');
+  await page.notebook.addCell('code', '2+2');
+  await page.notebook.addCell('markdown', '# Heading 2');
+  await page.notebook.addCell('code', '3+3');
+  await page.notebook.addCell('code', '4+4');
+}
+
+test.describe('Collapsible Headings; keyboard navigation', () => {
+  test.use({
+    mockSettings: {
+      '@jupyterlab/apputils-extension:themes': {
+        overrides: {
+          'content-font-family': 'Arial',
+          'content-font-size1': '12pt'
+        }
+      }
+    }
+  });
+
```

Comment:
Let's remove this in the final version and use the snapshot produced by the CI. The snapshots could then be all updated at once when changing the default font family.

```suggestion
```